doc/source/whatsnew/v2.1.0.rst

```diff
@@ -198,7 +198,7 @@ Sparse
 
 ExtensionArray
 ^^^^^^^^^^^^^^
--
+- Bug in :meth:`Series.any` and :meth:`Series.all` for empty or all null pyarrow-backed dtypes (:issue:`51624`)
```

Comment:
Can you explain what the bug was? Had to run an example to understand it. Otherwise lgtm